skins/fries.scss

```
@@ -0,0 +1,10 @@
+.example {
```

The css selector should similar to the file name.
```suggestion
.fries {
```